### ISSUE TITLE
Input calendar icon not visible in Chrome

### ISSUE THREAD

**Issue by DebugTheCode** [Wikiki/bulma-calendar#272]
## Bug Report

**Environment**
- Extension version(s): 6.1.7
- Bulma version: 0.9.2
- OS: OSX 11.2.2
- Browser: Chrome (91.0.4472.106)
- How you are customizing the extension: no

**Current Behavior**
The icon next to the startDate visible in the Input is not displayed.

```js
bulmaCalendar.attach('#' + id, {
            type: 'date',
            isRange: false,
            dateFormat: 'dd-MM-yyyy',
            cancelLabel: 'Annuleer',
            todayLabel: 'Vandaag',
            showClearButton: false,
            startDate: inputValue ? new Date(inputValue) : null,
            lang: 'nl'
        });
```

**Possible Solution**
I fixed it to put `-webkit-mask-image` additionally to `.datetimepicker-dummy .datetimepicker-dummy-wrapper:before, .datetimepicker-dummy .datetimepicker-dummy-wrapper::before` 's style.

**Additional context/Screenshots**

![e5fb01e0b0670dcaf8927eae24fe583e](https://user-images.githubusercontent.com/13984492/123280272-f2821c80-d508-11eb-8825-ce1d70632c66.png)

**Comment by michael-hack** [comment]
Works fine here in Chrome, Firefox and Edge. Could it possibly be related to your Bulma styles?
Do you also get the error in the [documentation](https://doc.mh-s.de/bulma-calendar/demonstration/date/)?

**Comment by DebugTheCode** [comment]
@michael-hack No, I don't see it happening on the documentation site. Should've look at that first. It's probably some misconfiguration or wrong version. I'll let you know what it was when I've fixed it.

**Comment by DebugTheCode** [comment]
@michael-hack I see the code on the documentation does has the `-webkit` prefix on `mask`. I'll have a look into this...

**Comment by shaundodimead** [comment]
I think this issue still persists, we had to fix by adding the webkit mask to the bulma source, but of course next fresh build it gets lost. What was the thinking on closing the issue?

**Comment by michael-hack** [comment]
Ok, thank you for your feedback. I couldn't reproduce the issue here and it looked like it is working on the documentation page.  We therefore assumed that it was a single individual error.

I' will look at this again.

**Comment by michael-hack** [comment]
Fixed in 6.1.10
